controllers/release/adapter_test.go

```diff
@@ -1236,13 +1236,24 @@ var _ = Describe("Release adapter", Ordered, func() {
 		})
 
 		When("the release has the automated label", func() {
+			AfterEach(func() {
+				_ = adapter.client.Delete(ctx, adapter.release)
+			})
+
 			BeforeEach(func() {
+				adapter = createReleaseAndAdapter()
 				adapter.release.Labels = map[string]string{
 					metadata.AutomatedLabel: "true",
 				}
+				adapter.ctx = toolkit.GetMockedContext(ctx, []toolkit.MockData{
+					{
+						ContextKey: loader.ReleasePlanContextKey,
+						Resource:   releasePlan,
+					},
+				})
 			})
 
-			It("returns invalid and an error if automated label is present but is not set in release status", func() {
+			It("returns invalid if automated label is present but is not set in release status", func() {
```

Comment:
why change this string? It returns `&controller.ValidationResult{Err: err}` so it has an error

Reply:
Yeah, my bad.